### PR TITLE
[New Rule] Persistence with Startup Folder by Unsigned Process

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/650

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
image

File: rules/windows/persistence_startup_folder_file_written_by_unsigned_process.toml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2020/11/29"
+maturity = "production"
+updated_date = "2020/11/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies files written or modified in the Startup Folder by unsigned processes. Adversaries may use this
+technique to maintain persistence.
```
Comment (bm11100) [inline rules/windows/persistence_startup_folder_file_written_by_unsigned_process.toml:10]:
```suggestion
Identifies files written or modified in the startup folder by unsigned processes. Adversaries may abuse this
technique to maintain persistence in an environment.
```

Comment (bm11100) [reply rules/windows/persistence_startup_folder_file_written_by_unsigned_process.toml:10]: may need a re lint

File: rules/windows/persistence_startup_folder_file_written_by_unsigned_process.toml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Persistence with Startup Folder by Unsigned Process"
```
Comment (bm11100) [inline rules/windows/persistence_startup_folder_file_written_by_unsigned_process.toml:15]:
```suggestion
name = "Startup Folder Persistence via Unsigned Process"
```

**Review by bm11100 — APPROVED** [review]

File: rules/windows/persistence_startup_folder_file_written_by_unsigned_process.toml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                         "C:\\Windows\\Temp\\*.exe", 
+                         "C:\\Windows\\Tasks\\*.exe", 
+                         "C:\\Intel\\*.exe", 
+                         "C:\\PerfLogs\\*.exe"
+                        )
+   ]
+   [file where event.type != "deletion" and user.domain != "NT AUTHORITY" and
+    file.path : ("C:\\Users\\*\\AppData\\Roaming\\Microsoft\\Windows\\Start Menu\\Programs\\Startup\\*", 
+                 "C:\\ProgramData\\Microsoft\\Windows\\Start Menu\\Programs\\StartUp\\*")
+   ]
```
Comment (brokensound77) [inline rules/windows/persistence_startup_folder_file_written_by_unsigned_process.toml:38]:
```suggestion
sequence by host.id, process.entity_id with maxspan=5s
  [process where event.type in ("start", "process_started") and process.code_signature.trusted == false and
  /* suspicious paths can be added here  */
   process.executable : ("C:\\Users\\*.exe", 
                         "C:\\ProgramData\\*.exe", 
                         "C:\\Windows\\Temp\\*.exe", 
                         "C:\\Windows\\Tasks\\*.exe", 
                         "C:\\Intel\\*.exe", 
                         "C:\\PerfLogs\\*.exe")
   ]
   [file where event.type != "deletion" and user.domain != "NT AUTHORITY" and
    file.path : ("C:\\Users\\*\\AppData\\Roaming\\Microsoft\\Windows\\Start Menu\\Programs\\Startup\\*", 
                 "C:\\ProgramData\\Microsoft\\Windows\\Start Menu\\Programs\\StartUp\\*")
   ]
```

File: rules/windows/persistence_startup_folder_file_written_by_unsigned_process.toml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+maturity = "production"
+updated_date = "2020/11/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies files written or modified in the startup folder by unsigned processes. Adversaries may abuse this
+technique to maintain persistence in an environment.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
```
Comment (brokensound77) [inline rules/windows/persistence_startup_folder_file_written_by_unsigned_process.toml:12]: would need to drop `winlogbeat-*` due to `process.code_signature.trusted`

**Review by brokensound77 — APPROVED** [review]
A couple comments then LGTM 👍

**Pull request merged**